Tests/SwiftParserTest/translated/MultilineErrorsTests.swift

```diff
@@ -588,7 +600,11 @@ final class MultilineErrorsTests: XCTestCase {
       diagnostics: [
         DiagnosticSpec(locationMarker: "1️⃣", message: "invalid escape sequence in literal"),
         DiagnosticSpec(locationMarker: "2️⃣", message: #"expected '"""' to end string literal"#, fixIts: [#"insert '"""'"#]),
-      ]
+      ],
+      fixedSource: ##"""
+        _ = """
+          foo\"""
```

Comment:
`"""` should be inserted on a new line.

Reply:
https://github.com/apple/swift-syntax/issues/1603